docs/faq.md

```diff
@@ -64,6 +64,18 @@ In some cases, there may be no applicable versions, so the `affected.versions[]`
 array is empty. This field, when empty, is omitted in the API output, and
 present (but empty) in the [data exports](#is-the-database-available-to-download).
 
+## How does OSV.dev handle withdrawn records?
+
+Records that have the [`withdrawn`](https://ossf.github.io/osv-schema/#withdrawn-field) field set will be excluded from the responses of POST API queries and from the main [list page](https://osv.dev/list).
```

Comment:
This might read a bit better and more comprehensively

```suggestion
Records that have the [`withdrawn`](https://ossf.github.io/osv-schema/#withdrawn-field) field set will be excluded from:
* the responses to POST API queries
* the main [list page](https://osv.dev/list) and related search results
```